opc/resource_lbaas_server_pool.go

```diff
@@ -0,0 +1,406 @@
+			},
+			"health_check": {
+				Type:     schema.TypeList,
+				Optional: true,
+				MaxItems: 1,
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"accepted_return_codes": {
+							Type:     schema.TypeList,
+							Optional: true,
+							Computed: true,
+							Elem:     &schema.Schema{Type: schema.TypeString},
+						},
+						"enabled": {
+							Type:     schema.TypeBool,
+							Optional: true,
+							Default:  true,
+						},
+						"healthy_threshold": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							// Computed: true,
+							Default:      5,
+							ValidateFunc: validation.IntBetween(2, 10),
+						},
+						"interval": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							// Computed: true,
+							Default:      30,
+							ValidateFunc: validation.IntBetween(5, 300),
+						},
+						"path": {
+							Type:     schema.TypeString,
+							Optional: true,
+							// Computed: true,
+							Default: "",
+						},
+						"timeout": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							// Computed: true,
+							Default:      20,
+							ValidateFunc: validation.IntBetween(2, 60),
+						},
+						"type": {
+							Type:     schema.TypeString,
+							Optional: true,
+							Default:  "http",
+							ValidateFunc: validation.StringInSlice([]string{
+								"http",
+							}, true),
+						},
+						"unhealthy_threshold": {
+							Type:     schema.TypeInt,
+							Optional: true,
+							// Computed: true,
+							Default:      7,
+							ValidateFunc: validation.IntBetween(2, 10),
+						},
+					},
+				},
+			},
+			"servers": {
+				Type:     schema.TypeSet,
+				Required: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"vnic_set": {
+				Type:         schema.TypeString,
+				Optional:     true,
+				ValidateFunc: validateComputeResourceFQDN,
+			},
+			"tags": {
+				Type:     schema.TypeSet,
+				Optional: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+
+			// Read only attributes
+			"consumers": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"operation_details": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+			"state": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+			"status": {
+				Type:     schema.TypeBool,
+				Computed: true,
+			},
+			"uri": {
+				Type:     schema.TypeString,
+				Computed: true,
+			},
+		},
+	}
+}
+
+func resourceOriginServerPoolCreate(d *schema.ResourceData, meta interface{}) error {
+	lbaasClient, err := meta.(*Client).getLBaaSClient()
+	if err != nil {
+		return err
+	}
+	serverPoolClient := lbaasClient.OriginServerPoolClient()
+
+	var lb lbaas.LoadBalancerContext
+	if loadBalancer, ok := d.GetOk("load_balancer"); ok {
+		lb = getLoadBalancerContextFromID(loadBalancer.(string))
+	}
+
+	input := lbaas.CreateOriginServerPoolInput{
+		Name: d.Get("name").(string),
+	}
+
+	if enabled, ok := d.GetOk("enabled"); ok {
```

Comment:
As noted above, we can just set this in the input since it has a default

Reply:
done